index-test.js

```diff
@@ -1,20 +1,59 @@
-/*
+// MINIMUM requirement of Unit Tests (as stated on project brief)
 
-MINIMUM requirement of Unit Tests (as stated on project brief)
+//test("Submitting a new task adds it to the list", () => {
+// test goes here
+//});
 
-test("Submitting a new task adds it to the list", () => {
-  // test goes here
-});
+setTimeout(() => {
 
-test("Checking an entry marks it as complete", () => {
-  // test goes here
-});
+  test("Checking first item marks it as complete", () => {
+  const checkbox = document.querySelectorAll("input[type='checkbox']")
+  const title = document.querySelectorAll('.movie-title');
+  checkbox[0].click();
+  const result = title[0].classList.contains('strikethrough'); 
+  const expected = true;
+  equal(result, expected)
+  checkbox[0].click(); //undoing the click
+  });
+
+  test("Checking third item marks it as complete", () => {
+  const checkbox = document.querySelectorAll("input[type='checkbox']")
+  const title = document.querySelectorAll('.movie-title');
+  checkbox[2].click();
```

Comment:
This is very clear to read that this is testing the third movie title in the list once the user clicks, brilliant work 🍿 📺